### PR TITLE
Adds blocker at mobile sizes to warn users that Bordo isn't mobile-fr…

### PR DESCRIPTION
**Pull request by michaelpanik**
…iendly

### CODE REVIEW

File: lib/bordo_web/templates/layout/client_app.html.leex [hunk 1]
```diff
@@ -4,7 +4,13 @@
 
 <%= live_component(@socket, BordoWeb.Components.Toast, id: "toast", show: false) %>
 
-<div class="h-screen flex">
+<div class="md:hidden bg-blue-100 h-screen p-8 flex flex-col items-center justify-center text-center">
+  <img src="/images/logo.svg" class="mx-auto mb-5" alt="Bordo admin" width="120px" />
```
Comment (chevinbrown) [inline lib/bordo_web/templates/layout/client_app.html.leex:8]: Let's use `static_path` here?
```suggestion
  <img src="<%= Routes.static_path(@socket, "/images/logo.svg") %>" class="mx-auto mb-5" alt="Bordo admin" width="120px" />
```

**Review by chevinbrown — APPROVED** [review]
Just check asset-paths & rebase for the green ✅

File: lib/bordo_web/templates/layout/client_app.html.leex (hunk 1 above)
Comment (michaelpanik) [reply lib/bordo_web/templates/layout/client_app.html.leex:8]: Should have probably asked about this, but that didn't work, error said essentially that @socket wasn't in scope, I believe. Or maybe I did something else wrong. Let me try this locally and report back in.

Comment (michaelpanik) [reply lib/bordo_web/templates/layout/client_app.html.leex:8]: Oops, no, I was using `@conn` instead of `@socket`. Fixing, rebasing, pushing.

**Comment by michaelpanik** [comment]
@chevinbrown I'm still misunderstanding something about the formatting, I guess

**Comment by chevinbrown** [comment]
> @chevinbrown I'm still misunderstanding something about the formatting, I guess

If you rebase, it'll pull the proper changes in.
You can run the failing check locally too to see. `mix format --check-formatted`

**Comment by chevinbrown** [comment]
Cherry-picked these to master.

**Pull request closed without merging**